src/Testing/e2e/browse.cy.js

```diff
@@ -11,8 +11,8 @@ describe('browseProjectsSpec', { retries: 3 }, () => {
     var testNostrId = 'npub1upnev99qsx4haf0yq8jtkny8yr45c94yw42g7rkdgr4ewgv3sh2shmuxnk'
     it('browseBasic', () => {
         cy.clickOnNavBar(Navbar.BROWSE)
-        cy.get(`[data-cy=${BROWSE_DATA_CY.PROJECT_GRID}]`).contains(ERROR_MESSAGES.NO_PROJECTS_FOUND)
-        cy.clickElementWithDataCy(BROWSE_DATA_CY.FIND_BUTTON)
+        // cy.get(`[data-cy=${BROWSE_DATA_CY.PROJECT_GRID}]`).contains(ERROR_MESSAGES.NO_PROJECTS_FOUND)
```

Comment:
@itailiors check this out please

Reply:
it seems fine, 
@miladsoft correct me if im wrong, it seems that you fetch automatically when you load the browse page.
(I can specify the cypress test for that when PR is merged)
 also i think that whenever you search a project that does not exist (for example 'aaaaaa') you should be presented with something that tells you that the project was not found (like the element that fetch projects had that you removed)   
 
![image](https://github.com/block-core/angor/assets/78041027/4c48781d-939f-4f42-9025-9e0059f8d3c0)

Reply:
Yes . You are right . I will add it.